Examples/Titanium.Web.Proxy.Examples.Wpf/MainWindow.xaml.cs

```diff
@@ -88,7 +113,9 @@ public MainWindow()
             proxyServer.TunnelConnectResponse += ProxyServer_TunnelConnectResponse;
             proxyServer.ClientConnectionCountChanged += delegate { Dispatcher.Invoke(() => { ClientConnectionCount = proxyServer.ClientConnectionCount; }); };
             proxyServer.ServerConnectionCountChanged += delegate { Dispatcher.Invoke(() => { ServerConnectionCount = proxyServer.ServerConnectionCount; }); };
-            proxyServer.Start();
+            
+                proxyServer.Start(); 
+        
```

Comment:
Please remove the unnecessary empty lines from everywhere. This comment is only an example.
- The code should not have 2 consecutive empty lines
- Do not put empty lines in the beginning of a block (after "{")

Please fix the indent, `proxyServer.Start()` should be moved left by 4 spaces